pkg/kubelet/container/runtime.go

```diff
@@ -66,6 +66,8 @@ type Runtime interface {
 	// Attaches the processes stdin, stdout, and stderr. Optionally uses a
 	// tty.
 	ExecInContainer(container api.Container, pod *api.Pod, stdin io.Reader, stdout, stderr io.WriteCloser, tty bool)
+	// Forward the specified port from the specified pod to the stream.
+	PortForward(pod *api.Pod, port uint16, stream io.ReadWriteCloser)
```

Comment:
That's a good point. We need to standardize on one or the other. Right now everything in the interface takes api.Pod even when it doesn't need to. We can start with this one.

Changed it to Pod.